api/lib/application/usecases/checkUserCanAccessCampaign.js

```diff
@@ -0,0 +1,15 @@
+const userRepository = require('../../infrastructure/repositories/user-repository');
+const campaignRepository = require('../../infrastructure/repositories/campaign-repository');
+const _ = require('lodash');
+
+module.exports = {
+
+  execute(userId, campaignId) {
```

Comment:
Pour exprimer l'intention 
```suggestion
  userBelongsToCampaingOrganization(userId, campaignId) {
```
Ou avec une IIFE (mais bien indentée)
`return (function userBelongsToCampaingOrganization(){ return _.some(user.memberships, { organization: { id : campaign.organizationId } });})()`

Reply:
Le nom du fichier ne te semble pas suffisant pour expliquer ce que va faire la fonction 'execute' ? J'ai repris le fonctionnement des autres usecases appelés par le securityController

Reply:
Je pense que le nommage de la fonction execute, est lié au pattern execute.